modules/strap/strap.service.js

```diff
@@ -123,8 +118,7 @@ class StrapService {
 
     await uploadService.deleteFiles(image);
 
-    const uploadImage = await uploadService.uploadSmallImage(image);
-    strap.image = uploadImage.fileNames.small;
```

Comment:
Delete this because this action provides to error. Function uploadSmallImage" from '../../utils/constructorOptionCommonFilters' return not object, but only string for small image.